jabel-javac-plugin/src/main/java/com/github/bsideup/jabel/PreviewFeatureCheckOverride.java

```diff
@@ -0,0 +1,24 @@
+package com.github.bsideup.jabel;
+
+import com.sun.tools.javac.code.Preview;
+import com.sun.tools.javac.code.Source;
+import net.bytebuddy.asm.Advice;
+
+public class PreviewFeatureCheckOverride {
```

Comment:
Let's follow the convention of `${ClassBeingVisited}Advice`:

```suggestion
public class PreviewAdvice {
```